content/articles/react-flux-frontend.md

```diff
@@ -0,0 +1,65 @@
+
+In this article we are going to have a look at my preferred set of tools for writing React/Flux webapps.
+If you don't know React or Flux, please look at their respective websites: [React](http://facebook.github.io/react/), [Flux](https://facebook.github.io/flux/).
+<!-- PELICAN_END_SUMMARY -->
+
+
+## Tools of the trade
+The React community is embracing ES6 — and sometimes ES7 — through [Babel](http://babeljs.io/). I personally like Typescript but recently decided to go with Babel for a few of reasons:
+
+- a constant changes in libraries and their sheer number were making it very hard to keep up-to-date definitions
+- Typescript is still lacking support for some things, making some libraries like [ImmutableJS](http://facebook.github.io/immutable-js/) and its records very hard to use
+- getting tired of casting things to <any> because of the compiler not picking up the correct things
+
+I do very much like types though, so I will have a look at [Flow](http://flowtype.org/) when it fully [supports ES6](https://github.com/facebook/flow/issues/560).
+
+You might also see the term isomorphic (confusing) or more recently universal, meaning that it can be rendered on the server as well.
+
+
+### Dev dependencies
+If you look at most React/flux boilerplates, you would notice they might use different tools than you might be used with Angular for example.
+
+
+Firstly, as mentioned in the previous section, Babel is omnipresent along with its linter of choice, [eslint](http://eslint.org/).
+
+
+A good number of projects eschew tasks runners such as [gulp](http://gulpjs.com/) in favour of a tool called [Webpack](http://webpack.github.io/) and npm tasks. Webpack is a module bundler like [Browserify](http://browserify.org/) but while Browserify takes the Unix philosophy of having small modules working together, Webpack comes with a lot more out of the box. Both works great though, so choose the one you like the most.
+
+
+One feature that made Webpack stand out early on with React was hot reloading using [react-hot-loader](https://github.com/gaearon/react-hot-loader) but is now available as [LiveReactload](https://github.com/milankinen/livereactload) for browserify.
+
+Testing wise, I have seen lots of repos using [jsdom](https://github.com/tmpvar/jsdom) which seems backwards to me as we can run tests in actual browsers using [Karma](http://karma-runner.github.io/0.13/index.html) for example.
+
+
+### Dependencies
+In terms of actual dependencies, we obviously need React. Note that since 0.14, you will also need the [react-dom](https://www.npmjs.com/package/react-dom) package to actually render.
+
+In terms of Flux implementation, my favourite is [Redux](http://rackt.github.io/redux/) which actually deviates of the Flux reference, leaning towards a FRP (look [here](https://gist.github.com/staltz/868e7e9bc2a7b8c1f754) for a nice introduction on reactive programming) approach like [Elm](http://elm-lang.org/) and having pure functions.
```

Comment:
"deviates from the Flux reference".

Can you make the FRP acronym explicit? Maybe this can be done by changing "look [here](https://gist.github.com/staltz/868e7e9bc2a7b8c1f754) for a nice introduction on reactive programming" to "[nice introduction on Functional Reactive Programming](https://gist.github.com/staltz/868e7e9bc2a7b8c1f754)"